### PR TITLE
add topology field for nvidia device plugin

### PR DESCRIPTION
**Pull request by carmark**
ref: https://github.com/kubernetes/kubernetes/pull/74423

### CODE REVIEW

File: nvidia.go [hunk 1]
```diff
@@ -29,6 +29,11 @@ func getDevices() []*pluginapi.Device {
 		devs = append(devs, &pluginapi.Device{
 			ID:     d.UUID,
 			Health: pluginapi.Healthy,
+			Topology: &pluginapi.TopologyInfo{
+				Node: &pluginapi.NUMANode{
```
Comment (klueska) [inline nvidia.go]: Thanks for this PR. Would you mind updating this to the newest API interface?

https://github.com/kubernetes/kubernetes/commit/1c1f19c61c18b7176674dce3ff9f64248fe74a89#diff-45056b94548d2ceef7f0d1028f937a05L77

**Comment by carmark** [comment]
@klueska done.

File: nvidia.go (hunk 1 above)
Comment (carmark) [reply nvidia.go]: done.

**Comment by klueska** [comment]
The change to add the `Topology` field looks OK to me, but I'm not 100% sure about the vendoring update.

/cc @RenaudWasTaken can you please take a look and verify that the vendoring patch

**Comment by RenaudWasTaken** [comment]
Hello!

Can you move your PR to : https://gitlab.com/nvidia/kubernetes/device-plugin
The code is now hosted there and mirrored to github!

**Comment by RenaudWasTaken** [comment]
Also it looks like you haven't signed-off your commits, do you mind taking care of it?
https://stackoverflow.com/questions/13043357/git-sign-off-previous-commits

Thanks a lot!

**Comment by RenaudWasTaken** [comment]
Note that the project was just relicensed to apache2: https://github.com/NVIDIA/k8s-device-plugin/blob/master/CONTRIBUTING.md

**Comment by carmark** [comment]
@RenaudWasTaken sure, no problem.

**Comment by carmark** [comment]
close, since https://github.com/NVIDIA/k8s-device-plugin/commit/cc0aad87285b573127add4e487b13434a61ba0d6

**Pull request closed without merging**